src/ts/Offer/Create/Inputs.tsx

```diff
@@ -30,6 +30,7 @@ const inputGenerator = (type, key, label) => {
       return (
         <div className="col">
           <FormsyText
+            id={`offer-create-${label}`}
```

Comment:
I don't really like using `id` tags, do we need them? Also, check out https://www.npmjs.com/package/react-html-id